addons/mail/static/tests/owl/test_utils.js

```diff
@@ -653,6 +654,39 @@ function pasteFiles(el, files) {
     el.dispatchEvent(ev);
 }
 
+function patchMessagingService(messaging_service, session = {}) {
```

Comment:
make it private, since used only in this file

Reply:
it's used in :
- `view_editor_manager_tests.js`
- `invoice_extract_form_view_tests.js`